### PR TITLE
Add TOC links to website

### PR DESCRIPTION
**Pull request by sdake**
The TOC governance material is very hard to find. Call it out clearly as
a top level item in the about pages. In the future, it may make sense to
transfer this document from istio/community directly to the website for
publishing after some post-processing.

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/about/community/toc/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Technical Oversight Committee
+description: Information describing the technical oversight committee within the Istio Community
+weight: 10
```
Comment (ericvn) [inline content/en/about/community/toc/index.md:4]: This weight is indeterminate with `Join`.

Comment (sdake) [reply content/en/about/community/toc/index.md]: thanks - resolved.

File: content/en/about/community/toc/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: Technical Oversight Committee
+description: Information describing the Istio technical oversight committee.
```
Comment (ericvn) [inline content/en/about/community/toc/index.md:3]: Here TOC is not capitalized. It is capitalized in the text below.

File: content/en/about/community/toc/index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+weight: 12
+keywords: [community]
+icon: community
+---
+Istio's Technical Oversight Community's [charter](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md#charter) and [membership](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md#committee-members).
```
Comment (ericvn) [inline content/en/about/community/toc/index.md:8]: Community vs Committee?  We seem to have multiple instances of both.

Comment (sdake) [reply content/en/about/community/toc/index.md]: ya, this should be fixed :).

File: content/en/about/community/toc/index.md (hunk 2 above)
Comment (sdake) [reply content/en/about/community/toc/index.md:3]: descriptiion as per website should be lower case should it not? If not, that's fine, I can modify it :)

Cheers,
-steve

**Comment by frankbu** [comment]
@craigbox PTAL. How does this fit with the reorg of this information you are planning?

**Comment by craigbox** [comment]
We're about to look into this as a whole, but I believe that istio.io is for users, and GH istio/community is for contributors. I see no reason to expose how the sausage is made on the marketing website.

Steve, what problem were you trying to solve?  I could see a high-level page talking about the governance of the project, including Steering, being part of the upcoming revision.

**Comment by sdake** [comment]
@craigbox the problem to be solved is that the membership nor charter of the TOC nor steering is known publically unless one undertakes a serious hunt. Just being public information in the https://igithub.com/istio/community repo is not sufficient. The information must also be communicated widely - or we may appear less legitimate than we are as a community.

I think putting these two PRs on pause, because something may(?) be in the works in the future, doesn't make a lot of sense. If something does replace these pages, then we can certainly remove them at this time.

Acknowledging we have a charter and membership and a structure is part of running a public project. While I don't believe in lemming-think, may I point you at the following major open source organizations:

https://www.debian.org/devel/tech-ctte
https://www.linuxfoundation.org/about/board-members/
https://osf.dev/about/board/
https://governance.openstack.org/tc/reference/charter.html

I could continue with hundreds of examples, where Istio stands out as not following generally accepted practices, but I think the above list should be sufficient.

Cheers,
-steve

**Comment by craigbox** [comment]
As I said, I think it's entirely appropriate to have a "governance" page that introduces the concepts of a steering committee and a technical oversight committee, why they both exist and why they're different, and links off to the GitHub content. The two-links-per-page PRs don't give that context.  

I'm happy to collaborate on a draft, but don't have time for a couple of days.

I think a good outline would be:

- Groups

**Comment by frankbu** [comment]
I agree this stuff should be easier to find. It seems strange to me that the information is so hard to find given that it is already located where you'd expect (About -> Community).

It's at the "how we do things" link:

image

The weird thing is that entering "TOC", "Technical Oversight", "Steering Committee" doesn't return anything useful.

**Comment by sdake** [comment]
> As I said, I think it's entirely appropriate to have a "governance" page that introduces the concepts of a steering committee and a technical oversight committee, why they both exist and why they're different, and links off to the GitHub content. The two-links-per-page PRs don't give that context.
> 
> Consider the primary user of istio.io: someone who wants to use the software, but has no idea about how it is made. Every choice I make is through that lens. We want to surface the fact that this information exists, and that they can go somewhere else to find it, but an average user doesn't need this. Debian, for example, has theirs listed under "Developer's Corner". We try and make "community" do the heavy lifting for both users and developers at this time.
> 
> I'm happy to collaborate on a draft, but don't have time for a couple of days.
> 
> I think a good user-focused outline would be:
> 
> * The two groups
>   
>   * Why they're different
> * Steering
>   
>   * Responsibilities
>     
>     * Advocacy
>     * Ecosystem
>   * Community seats
>   * Contribution seats
> * TOC
>   
>   * Make up
>     
>     * Even number, drive consensus
>   * Responsibilities
>   * Consensus
> * Working Groups
>   
>   * Code ownership
>   * Leads
> * Code of Conduct

@craigbox - I have a simple proposal to resolve this disagreement. How about @frankbu merge my two PRs, and when you feel you have the time to implement your proposal, you can so without being blocked. Your proposal is reasonable for the website, however, it is an unreasonable justification to hold up merging this work.

If you can provide a competitive PR that solves the same problem, I am happy to do the work of reverting my PRs and providing proper aliases in your PRs as a follow-up.

Cheers,
-steve

**Comment by sdake** [comment]
@frankbu thanks - I wasn't aware those links were there. 

Cheers,
-steve

**Comment by frankbu** [comment]
Superseded by #8382

**Pull request closed without merging**